src/PowerShellEditorServices.Protocol/MessageProtocol/Channel/StdioServerChannel.cs

```diff
@@ -27,11 +27,8 @@ public StdioServerChannel(ILogger logger)
 
         protected override void Initialize(IMessageSerializer messageSerializer)
         {
-#if !CoreCLR
```

Comment:
This change might not be done the best way -- perhaps we should test the PowerShell version here?